zipline/testing/fixtures.py

```diff
@@ -359,14 +415,19 @@ class WithInstanceTmpDir(object):
     with a `testfixtures.TempDirectory` object whose path is
     `cls.TMP_DIR_PATH`.
 
-    The default value of TMP_DIR_PATH is None which will create a unique
-    directory in /tmp.
+    Attributes
+    ----------
+    INSTANCE_TMP_DIR_PATH : str
+        The path to the new directory to create. By default this is None
+        which will create a unique directory in /tmp.
     """
-    TMP_DIR_PATH = None
+    INSTANCE_TMP_DIR_PATH = None
 
     def init_instance_fixtures(self):
         super(WithInstanceTmpDir, self).init_instance_fixtures()
-        self.tmpdir = self.enter_instance_context(tmp_dir(self.TMP_DIR_PATH))
+        self.instance_tmpdir = self.enter_instance_context(
+            tmp_dir(self.TMP_DIR_PATH),
+        )
 
 
 class WithBcolzDailyBarReader(WithTmpDir, WithSimParams):
```

Comment:
It seems odd to me that this inherits from `WithSimParams`.  That to me implies that you need a sim params to create a daily bar reader, which isn't true.